### PR TITLE
adding a new connector for accessing rss & atom feeds

### PR DESCRIPTION
**Pull request by nickski15**
I've written a new community connector that connects to both RSS and ATOM feeds. I would like to contribute it to the data studio community.

### CODE REVIEW

**Comment by nickski15** [comment]
I've signed it

File: community-connectors/rss-atom-feeds/main.js [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+ * @param {Array.<Field>} requestFields An array of Field objects that
+ *     only have the Field.name property set.
+ * @param {object} allFields An object that has all properties of the Field
+ *     object set. The keys of the object are Field.names and the values of
+ *     the object are the entire Field objects.
+ * @return {Array.<Field>} An array of Field objects.
+ */
+function getRequestedFields(requestFields, allFields) {
+  var fields = [];
+  for (var i = 0, requestField; requestField = requestFields[i]; ++i) {
```
Comment (diminishedprime) [inline community-connectors/rss-atom-feeds/main.js]: This iteration is difficult to follow. Could you just use a `.forEach` instead?

File: community-connectors/rss-atom-feeds/main.js [hunk 2]
```diff
@@ -0,0 +1,321 @@
...
+  dataType: STRING,
+  semantics: {
+    conceptType: DIMENSION,
+    semanticType: YEAR_MONTH_DAY_HOUR
+  }
+};
+
+/**
+ * Synthetic fields that are derived from the fields above.
+ * Data Studio will derived these on the fly and should be
```
Comment (diminishedprime) [inline community-connectors/rss-atom-feeds/main.js]: Phrasing here is confusing. Could you reword?

**Comment by diminishedprime** [comment]
closing due to inactivity. Feel free to open up a new one if you're still interested in getting this merged in!

**Pull request closed without merging**